### ISSUE TITLE
Duplicate definition warnings when using contiguous IDL

### ISSUE THREAD

**Issue by inexorabletash** [w3c/respec#480]
example:

```html
<pre class="idl">
interface Foo {
  attribute DOMString attr;
};
</pre>
<p>The <dfn data-lt="Foo.attr">attr</dfn> attribute's getter must...</p>
```
Yields: `ERROR: Duplicate definition of 'foo/attr'`

Note that the id/href do end up being created correctly (in this case, `id="dom-foo-attr"`).

**Comment by marcoscaceres** [comment]
@inexorabletash, try the following:

```HTML
<p link-for="Foo">The <dfn data-lt="Foo.attr">attr</dfn> attribute's getter must...</p>
```

Should hopefully work. There is some good guidance here that should hopefully fix all the issues in the IDBlv2 spec (which I was not fully aware of):

https://www.w3.org/respec/examples/webidl-contiguous.html#introduction

**Comment by marcoscaceres** [comment]
(please reopen if still busted or other issues/questions come up)

**Comment by inexorabletash** [comment]
After the fixes were made to #470 this became a non-issue. The IDB spec is good now. :)
